### ISSUE TITLE
 I'm having problems with this: "nextPageToken"

### ISSUE THREAD

**Issue by mathewvieira** [sayok88/likedmusicyoutube#1]
```
Traceback (most recent call last):
  File "main.py", line 113, in <module>
    main()
  File "main.py", line 88, in main
    destination_vids = getplaylistvids(destination_play_list)
  File "main.py", line 59, in getplaylistvids
    nextPageToken = response['nextPageToken']
KeyError: 'nextPageToken'
```
Any suggestion?

**Comment by sayok88** [comment]
I will take a look at that and come back to you.

**Comment by mathewvieira** [comment]
okay, I couldn't find anything but I'm also not very good with python, so...

**Comment by mathewvieira** [comment]
News? @sayok88

**Comment by mawoka-myblock** [comment]
Experienced the same problem

**Comment by mawoka-myblock** [comment]
Fixed in #2 (i think) @mathe8

**Comment by mathewvieira** [comment]
@mawoka-myblock, Thank you! I will test it as soon as possible and give you a feedback.

**Comment by mawoka-myblock** [comment]
Nice! Added also a new way to configure everything. Pls give me feedback about it.

**Comment by mathewvieira** [comment]
I didn't quite understand the new documentation. Is there a request limit for 200 songs? @mawoka-myblock

**Comment by mawoka-myblock** [comment]
There is a limit of actions you can perform per day over the YouTube-Api. So after it moved 200 Songs I used all of my quota. I have to wait for the next day untill the quota gets ressetet.

**Comment by mawoka-myblock** [comment]
btw it worked pretty well with answering an email!

**Comment by mathewvieira** [comment]
Hello, @mawoka-myblock, 

**I tested the fix and it is working correctly😃**, just some steps that I was in doubt... but this is my fault haha.

Here are the topics : 

1. I have an functionality idea that could be interesting. Be able to export links from LM playlist to a txt file.
I believe that this would not consume the Youtube API quota. It would be possible?

2. In google cloud, there is a way to increase the amount of quota points or API requests? 
(for the YouTube API in particular) Or some method of contour this consumption?

**Comment by mawoka-myblock** [comment]
You can ask Google for a greater quota but I didn't try that because you would have to fill in a form where you have to specify the reason for a greater quota. You can try that but I didn't.

**Comment by mawoka-myblock** [comment]
About the text-file: I think the problem isn't to get the songs. The problem is that it writes song for song into a new playlist which you cannot make more efficient I think.

**Comment by mawoka-myblock** [comment]
I also didn't understand the code completely so I think I won't be able to add features like this but maybe in the future. I mean the script is in working condition

**Comment by mathewvieira** [comment]
Good news @mawoka-myblock, I created a code that does this export process (basically I just made some changes to main.py and added a few lines of code). I will soon create a pull request with this functionality.

**Comment by mathewvieira** [comment]
@mawoka-myblock, take a look at the new pull request #3

**Comment by mawoka-myblock** [comment]
Could you create the PR on my Repo because I thnink @sayok88 isn't active anymore?

**Comment by mawoka-myblock** [comment]
@mathe8 What do you mean, should I create a web-app for this?

**Comment by mathewvieira** [comment]
@mawoka-myblock, It would be interesting, I do not understand very well about Web Applications with Python, 
but it can be a good opportunity to improve this project.

**Comment by mawoka-myblock** [comment]
The problem is againg the rate-limit for the api but maybe if I would program it Google would expand my quota

**Comment by mathewvieira** [comment]
@mawoka-myblock, I can't think of a more practical solution than that you quoted.
My intention was to be able to use this application in parallel with : https://github.com/ytdl-org/youtube-dl
This quota limit is a major impediment. I think this 200 song limit is too low to transfer.
